src/arrayReverse.test.js

```diff
@@ -7,14 +7,29 @@ describe(`Function 'arrayReverse':`, () => {
     expect(arrayReverse).toBeInstanceOf(Function);
   });
 
-  it(`should return an array`, () => {
-
+  it(`should return an array of reversed worls`, () => {
+    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
```

Comment:
The expected result `['ymed', 'acAetaM']` does not match the task description. According to the task, the function should reverse the strings and their order, so the expected result should be `['yemed', 'etaMca']`.